odonto/admin.py

```diff
@@ -71,8 +74,34 @@ def performer_number(self, obj):
         return ''
 
 
+class PatientAdmin(VersionAdmin):
+    list_display = ('__str__', 'patient_detail_link')
+
+    search_fields = [
+        'demographics__first_name',
+        'demographics__surname',
+        'demographics__hospital_number'
+    ]
+
+    def patient_detail_url(self, obj):
+        return reverse('odonto-patient-detail', kwargs={"pk": obj.id})
+
+    def patient_detail_link(self, obj):
+        url = self.patient_detail_url(obj)
+        return format_html(
+            "<a href='{url}'>{url}</a>", url=url
```

Comment:
fstring?

Reply:
format_html does an implicit mark_safe otherwise you just end up with html on the page .

(I did check because I wasn't sure if this was true for the admin but it is)
<img width="269" alt="Screen Shot 2019-12-11 at 13 39 43" src="https://user-images.githubusercontent.com/2175455/70626130-c0a5af80-1c1b-11ea-8bbb-52ca28fc1756.png">


https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.html.format_html